### DISCUSSION TITLE
anyway to deacrease waiting time please?

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [mashb1t/Fooocus#21] [Q&A] [answered]
![image](https://github.com/mashb1t/Fooocus/assets/42430868/abf00107-3962-4e04-a1b0-982ee0e609ec)
perhaps its becuse im useing "main" branch and should use "hotfix/prevent-erroneous-image-grid-file-removal"?
Ty so much

**Comment by mashb1t** [comment, marked as answer]
The main branch is stable and had been tested to be working correctly. Please check the logs and your swap location or resource usage to identify bottnelecks.

**Reply by Amit30swgoh** [reply]
Will do thnks!
